src/Model/PowerElectronics/SynchronousMachine/SynchronousMachine.hpp

```diff
@@ -72,7 +72,7 @@ namespace ModelLib
         std::tuple<ScalarT,ScalarT> Rkq_;
         ScalarT Rfd_;
         ScalarT Rkd_;
-        ScalarT RJ_;
+        ScalarT Rjac_;
         ScalarT P_;
```

Comment:
Document variables using doxy syntax:
```suggestion
        ScalarT Rjac_; ///< Mysterious variable that is important for the simulations
        ScalarT P_; ///< Another mysterious variable
```

Reply:
There is no `RJ` in the model for synchronous machine, but based on the code, it seems this is rotor moment of inertia. Please comment the code accordingly.

Reply:
While I'm at it, I added explanations for the other variables (mostly based on Copilot suggestions). Let's see how it did.

Reply:
I choose `RJ` over `J` as `J` is commonly overloaded as a Jacobian (as seen by the previous change). It is the rotor moment of inertia. I left the variables as defined in the notation of the equations given.